runtime/container.go

```diff
@@ -22,7 +17,7 @@ type Container interface {
 	// Start starts the init process of the container
 	Start(checkpoint string, s Stdio) (Process, error)
 	// Exec starts another process in an existing container
-	Exec(string, specs.Process, Stdio) (Process, error)
+	Exec(string, processSpec, Stdio) (Process, error)
```

Comment:
This method now depends on a private type, it won't compile. 

Did you mean to have `runtime.ProcessSpec` instead?